File: Ruby/Queries/PrimeNumberChecker.rb
puts "Enter num to check if it is prime or not"
num = gets.to_i
temp = 2
prime = true
while temp < num 
    if num%temp == 0
        puts "Not Prime"
        prime = false
        break
    end
    temp += 1
end
if prime 
    puts "Prime Num"
end

File: Ruby/Queries/PrimeNumberChecker.c
#include <stdio.h>
#include <stdbool.h>
int main(){
    int num;
    scanf("%d",&num);
    int temp = 2;
    bool prime = true;
    while(temp < num){
        if(num%temp == 0){
            printf("Not Prime");
            prime = false;
            break;
        }
        temp += 1;
    }
    if(prime){
        printf("Prime Num");
    }
    return 0;
}